### ISSUE TITLE
Hashless `include` is not supported on Debian

### ISSUE THREAD

**Issue by nobodysu** [roddhjav/apparmor.d#5]
`include` is not supported on any Debian, not even bullseye. `#include`, however, is supported.
AFAIK, these were deprecated on Ubuntu (old one still works), but what about Arch? Maybe, `#include` should be used for better compatibility?

**Comment by roddhjav** [comment]
What error did you get. I use it on both Arch and Debian and both support `include` fine. `#include` is the old notation, but even debian 11 support the new.

**Comment by nobodysu** [comment]
```
# wget https://raw.githubusercontent.com/roddhjav/apparmor.d/master/apparmor.d/profiles-a-f/df -O /etc/apparmor.d/df
# aa-enforce df

ERROR: Syntax Error: Unknown line found in file /etc/apparmor.d/df line 26:
    include if exists <local/df> }
# lsb_release -a
No LSB modules are available.
Distributor ID:	Debian
Description:	Debian GNU/Linux 11 (bullseye)
Release:	11
Codename:	bullseye
```
It looks like `include if exists` is not supported by old tools.
But indeed, `systemctl restart apparmor` seems to work.

Can I submit my contributions with old includes and unique `_path` macro? At least while these issues are still present in the upstream?

**Comment by roddhjav** [comment]
As you mentioned use `systemctl restart apparmor`. You can also use `apparmor_parser`. Alternatively once you want to install all profile, you should directly build and install the Debian package as it will reload the AppArmor tool with systemd automatically. I might write some tooling to easily install and load a given profile without having to install the full package.

In general the profiles compilation works well as it is largely tested in the CI. For instance see the [last pipeline](https://gitlab.com/roddhjav/apparmor.d/-/pipelines/420241272) where on both distribution both the package build and the AppArmor preprocessing is tested.

You should submit contribution using the new style as there are over 1100 profiles that already use it and that compile well with it. If we allow both style it will be the mess soon.
So to sum up:
- Use `include` instead of `#include`
- Use the profile variable `@{exec_path}`
- For optional include use `include if exists`

**Comment by nobodysu** [comment]
Alright, got it.
